package.json

```diff
@@ -84,25 +93,5 @@
 			}
 		},
 		"timeout": "2m"
-	},
-	"xo": {
-		"extends": "xo-typescript",
-		"extensions": [
-			"ts"
-		],
-		"ignore": "legacy-tests",
```

Comment:
Is this still being ignored?

Reply:
No - the legacy tests have their own [eslint config](https://github.com/sequelize/umzug/blob/master/legacy-tests/.eslintrc.json) which is I believe pretty much what the project was using before your refactor. eslint is smart enough to handle hierarchical configs.